Library/Homebrew/cmd/audit.rb

```diff
@@ -512,9 +512,9 @@ def audit_github_repository
     end
 
     problem "GitHub fork (not canonical repository)" if metadata["fork"]
-    if (metadata["forks_count"] < 10) && (metadata["subscribers_count"] < 10) &&
-       (metadata["stargazers_count"] < 20)
-      problem "GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)"
+    if (metadata["forks_count"] < 50) && (metadata["subscribers_count"] < 50) &&
+       (metadata["stargazers_count"] < 100)
```

Comment:
> I'm not sure if we want to be so harsh on all other projects just because someone tries to game our system every once in a while (very rarely, I guess).

I more think: it it's easy enough to game, it's not a good notability indicator. I don't want Homebrew to be the source for a bunch of weekend projects.